src/components/Nav/Menu/components/Avatar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NoProfileAvatarIcon } from '@pancakeswap/uikit'
+import { Link } from 'react-router-dom'
+import { useProfile } from 'state/hooks'
+
+const StyledAvatar = styled.div`
+  margin-left: 8px;
+  position: relative;
+
+  img {
+    border-radius: 50%;
+  }
+`
+
+const Avatar: React.FC = () => {
```

Comment:
This component is already in UIKit (although its a bit more complicated over there)